FEXCore/Source/Interface/Context/Context.h

```diff
@@ -195,7 +195,7 @@ class ContextImpl final : public FEXCore::Context::Context {
   // returns false if a handler was already registered
   CustomIRResult AddCustomIREntrypoint(uintptr_t Entrypoint, CustomIREntrypointHandler Handler, void* Creator = nullptr, void* Data = nullptr);
 
-  void AppendThunkDefinitions(const fextl::vector<FEXCore::IR::ThunkDefinition>& Definitions) override;
+  void AppendThunkDefinitions(const std::span<FEXCore::IR::ThunkDefinition> Definitions) override;
```

Comment:
`const` has no effect for value parameters. Instead, the element references can be made const, since we copy them around anyway: `std::span<const ThunkDefinition>`

Reply:
changed